Sadot/GeneralClasses/DB/DBSQL.cs

```diff
@@ -531,7 +574,34 @@ public void UpdateTableStatus(int id, string status)
                 cmd.ExecuteNonQuery();
                 databaseConnection.Close();
             }
-            catch { }
+            catch (Exception ex)
+            {
+                MessageBox.Show(ex.Message);
+            }
+        }
+
+        /// <summary>
+        /// Method to update all parameters of specific table
+        /// </summary>
+        public void UpdateTableParams(int id, string status, string orderState, DateTime timeOfOrder)
+        {
+            string query = "UPDATE `tables` SET `tableStatus`= @status , `orderState`= @orderState , `timeOfOrder`= @timeOfOrder  WHERE `tableID` = @id";
+            MySqlCommand cmd = new MySqlCommand(query, databaseConnection);
+            cmd.CommandTimeout = 60;
+            cmd.Parameters.AddWithValue("@status", status);
+            cmd.Parameters.AddWithValue("@orderState", orderState);
+            cmd.Parameters.AddWithValue("@timeOfOrder", timeOfOrder);
+            cmd.Parameters.AddWithValue("@id", id);
+            try
+            {
+                databaseConnection.Open();
+                cmd.ExecuteNonQuery();
+                databaseConnection.Close();
+            }
+            catch (Exception ex)
+            {
+                MessageBox.Show(ex.Message);
```

Comment:
I merged all the functions to one that received the table and updated all its parameters.
your suggestion is a very good idea but it will take some time what do you say?